### DISCUSSION TITLE
image not showing

### DISCUSSION THREAD

**Discussion by RVP04** [spatie/laravel-pdf#45] [General]
my controller code
```
return Pdf::view('superadmin.reports.pdf.missing_students', [
            'reportTitle' => 'Missing Student Reports - Branch Wise',
            'missingStudents' => $missingStudents
        ])->headerView('layout.reports.header')
            ->footerView('layout.reports.footer')
            ->format('a4')
            ->name('Missing Students Name List');
```
header
```
<style>
    header {
        padding: 25px;
        float: right;
    }
    img
    {
        width: 175px;
        height: 75px;
    }
</style>
<header>
    <img src="{{ asset('img/gray-logo.png') }}"/>
</header>
```
but if i view normally it is showing

**Comment by ArielMejiaDev** [comment]
Try to build an absolute path, take in mind that it should be dynamically based on the environment where it is executed, eg:

```php
<style>
    header {
        padding: 25px;
        float: right;
    }
    img
    {
        width: 175px;
        height: 75px;
    }
</style>
<header>
    <img src="{{ config('base_url') . "/img/gray-logo.png" }}"/>
</header>
```

There should be a `img` directory inside `public` directory or check that you have a symlink with `php artisan storage:link` command, check that your implementation runs as a normal view, and try it using PDF.

**Reply by RVP04** [reply]
In normal view image showing correctly in a proper position

**Reply by RVP04** [reply]
@ArielMejiaDev still image not showing in header sir

**Reply by RVP04** [reply]
Still image not showing sir

**Reply by milandicic** [reply]
@RVP04 I had a similar issue and it was because I'm working on my local machine and I used Browsershot Lambda so my images were not available to my AWS Lambda instance over the internet. 

I'm not sure what is your current setup but it might happen that your images are not available to the puppeteer and Chromium instance. So to test it just try hardcoding one image that is available on the internet for example: https://picsum.photos/id/237/200/300 and see if it works.

**Reply by RVP04** [reply]
@milandicic @ArielMejiaDev @freekmurze still it is not showing. Kindly help me

**Reply by RVP04** [reply]
@freekmurze can you please check into this issue

**Comment by marinario** [comment]
The only way working for me was (tested only locally):
`<img src="{{base_path('/public/assets/img/image.png')}}">`
